src/servers/src/http.rs

```diff
@@ -546,7 +551,9 @@ impl HttpServer {
                     .layer(HandleErrorLayer::new(handle_error))
                     .layer(TraceLayer::new_for_http())
                     .layer(TimeoutLayer::new(self.options.timeout))
-                    .layer(DefaultBodyLimit::max(DEFAULT_BODY_LIMIT))
+                    .layer(DefaultBodyLimit::max(
+                        self.options.body_limit.0.try_into().unwrap(),
```

Comment:
not sure if `unwrap()` is suitable here... @evenyag

Reply:
yeah... I was not sure if I need to use `snafu` error during conversion. Happy to adjust if needed

Reply:
What about using the default value as a fallback? Casting DEFAULT_BODY_LIMIT to usize should be safe.
```suggestion
                        self.options.body_limit.0.try_into().unwrap_or_else(|| DEFAULT_BODY_LIMIT.as_bytes() as usize),
```